src/components/Buttons/AddToWishlist.jsx

```diff
@@ -0,0 +1,9 @@
+import './Buttons.css';
+
+export const AddToWishlist = () => {
+    return (
+        <button className="like-btn" onClick={() => { AddToWishlist() }}>
```

Comment:
here i m not able to understand you are calling the same the component

Reply:
removed this click